pinot-broker/src/test/resources/log4j2.xml

```diff
@@ -22,14 +22,15 @@
 <Configuration>
   <Appenders>
     <Console name="console" target="SYSTEM_OUT">
-      <PatternLayout>
-        <pattern>%d{HH:mm:ss.SSS} %c{1} - %m%n</pattern>
-      </PatternLayout>
+      <PatternLayout pattern="%d{HH:mm:ss.SSS} %p [%c{1}] [%t] %m%n"/>
```

Comment:
Do we have any kind of job that parses from the log outputs? (e.g. compatibility test)

I think that changing the pattern for test logs should be fine as long as we don't read anywhere in our testing jobs on Github action. But, we should be careful if it will change the production log pattern.